_data/banking.yml

```
@@ -498,18 +627,25 @@ websites:
       - hardware
       - sms
     doc: https://www.dnb.no/privat/kundeservice/innlogging-nettbank.html
+    regions:
+      - "no"
 
   - name: Dutch-Bangla Bank Limited (DBBL)
     url: https://www.dutchbanglabank.com/
     email_address: hrd_head@dbbl.com.bd
     img: dbbl.png
     lang: bn
+    regions:
+      - bd
+      - nl
```

```suggestion
    regions:
      - bd
```
DBBL [only serves Bangladesh](https://en.wikipedia.org/wiki/Dutch_Bangla_Bank).